text/core_rtos.tex

```diff
@@ -23,7 +23,8 @@ \section{Starting the system with the loader}
 This, in turn, enables multi-stage boot where some functionality, such as attestation, secure key storage, and so on, are provided by a bootloader.
 These abstractions can all be built from capabilities and so, unlike systems based on protection rings such as TrustZone, an arbitrary number can be nested.
 
-If a compartment contains a global that is a pointer, initialised to point to another global, the loader will initialise the pointer by deriving a capability from one out of the compartment's code or data capabilities.
+If a compartment contains a global that is a pointer, initialised to point to another global, the loader will initialise the pointer by deriving a capability from one of the compartment's code or data capabilities.
+\amanda{I don't think the "one of either" is necessary if we take out the "out".  However, as a reader, my immediate reaction is "how does it choose?" and it may be worthwhile to add the answer to that.}
```

Comment:
I think the new text implies that there may be more than one code or data capability for the compartment.  Possibly also add a sentence like:

> Globals may point to other globals owned by the compartment, the loader will fail to derive a valid capability if they point elsewhere.